workflow/controller/operator.go

```diff
@@ -191,14 +191,30 @@ func (woc *wfOperationCtx) operate() {
 
 	woc.log.Infof("Processing workflow")
 
-	// Load the WorkflowSpec for execution
-	execTmplRef, execArgs, err := woc.loadExecutionSpec()
+	// Loading the Execute workflow spec for execution
+	// ExecWF is a runtime execution spec which merged from Wf, WFT and Wfdefault
+	err := woc.loadExecWFFromWfDefaultOrWFTRef()
```

Comment:
This new function will load the ExecWF in Context. This function will make sure `storedworkflowSpec` set for `workflowtemplateRef` scenario and WorkflowDefault merged for normal workflow scenario